test/addons_test.go

```diff
@@ -0,0 +1,203 @@
+package test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/blang/semver"
+	"gopkg.in/yaml.v2"
+	"sigs.k8s.io/kind/pkg/apis/config/v1alpha3"
+	"sigs.k8s.io/kind/pkg/cluster"
+
+	"github.com/mesosphere/kubeaddons/hack/temp"
+	"github.com/mesosphere/kubeaddons/pkg/api/v1beta1"
+	"github.com/mesosphere/kubeaddons/pkg/catalog"
+	"github.com/mesosphere/kubeaddons/pkg/repositories/git"
+	"github.com/mesosphere/kubeaddons/pkg/repositories/local"
+	"github.com/mesosphere/kubeaddons/pkg/test"
+	"github.com/mesosphere/kubeaddons/pkg/test/cluster/kind"
+)
+
+const (
+	defaultKubernetesVersion = "1.15.6"
```

Comment:
Why are we using 1.15 instead of 1.16? (Even better would be if it was a parameter, but that can come later.)

Reply:
partially copy/paste issue, partially kindest/node doesn't stay current with k8s patches, either way, updating to latest 1.16 available:
```suggestion
	defaultKubernetesVersion = "1.16.4"
```